root/static/scripts/edit/externalLinks.js

```diff
@@ -259,6 +259,15 @@ export class ExternalLinksEditor
             return (
               <ExternalLink
                 errorMessage={error || ''}
+                handleUrlBlurCallback={
```

Comment:
I'd also remove `Callback` from all the names, since `handle` already indicates it's a handler, and callback doesn't really make sense combined with it (sounds like the handler is supposed to call it or something, but it's the handler itself).

Reply:
I think I fixed them all now. It does have handleUrlBlur binding this.handleUrlBlur, but that's probably fine?

Reply:
Yeah, looks better now.